tests/test_agent.py

```diff
@@ -67,3 +67,40 @@ async def test_info(agent):
         "config": {},
         "client_drop_p0s": True,
     }
+
+
+async def test_apmtelemetry(
```

Comment:
we could probably add an integration test with `ddtrace` (manually enable the telemetry writer) but can follow up with this.